packages/common/src/store/pages/chat/slice.ts

```diff
@@ -113,17 +143,28 @@ const slice = createSlice({
         chatId,
         response: { data, summary }
       } = action.payload
-      state.chatMessages[chatId].status = Status.SUCCESS
-      state.chatMessages[chatId].data =
-        state.chatMessages[chatId].data.concat(data)
-      state.chatMessages[chatId].summary = summary
+      chatsAdapter.updateOne(state.chats, {
+        id: chatId,
+        changes: {
+          messagesStatus: Status.SUCCESS,
+          messagesSummary: summary
+        }
+      })
+      chatsAdapter.updateOne(state.chats, {
+        id: chatId,
+        changes: { messagesStatus: Status.SUCCESS, messagesSummary: summary }
+      })
+      chatMessagesAdapter.upsertMany(state.messages[chatId], data)
```

Comment:
so does chatMessagesAdapter with upsertMany change the entire list of messages across all chats, or just the messages for this chat?

Reply:
just the messages for this chat :)